common/src/main/java/dev/ftb/mods/ftblibrary/api/sidebar/SidebarButton.java

```diff
@@ -0,0 +1,42 @@
+package dev.ftb.mods.ftblibrary.api.sidebar;
+
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+
+import java.util.List;
+import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
+
+public interface SidebarButton
+{
+
+    /**
+     * @return the id of the button
```

Comment:
You can likely just explain what this does instead of specifically noting what the return is